app/src/main/java/org/fossasia/openevent/app/utils/ViewUtils.java

```diff
@@ -7,15 +7,17 @@
 
 public class ViewUtils {
 
-    public static void showView(View view, boolean show) {
-        int mode = View.GONE;
-
+    public static void showInvisibleView(View view, int mode, boolean show) {
```

Comment:
Please remove mode from the method name as it is taking mode in parameter. why not same name?